tests/conditional_compilation/conftest.py

```diff
@@ -7,9 +7,9 @@
 """ Pytest configuration for compilation tests.
 
 Sample usage:
-python3 -m pytest --artifacts ./compiled --test_conf=<path to test config> \
+python3 -m pytest --test_conf=<path to test config> \
     --sea_runtool=./IntelSEAPI/runtool/sea_runtool.py \
-    --benchmark_app=./bin/benchmark_app test_collect.py
+    --benchmark_app=./bin/benchmark_app --artifacts ./compiled test_collect.py
```

Comment:
mb infer_tool?

Reply:
We can calculate a path to the tool similar to a code on line 24. Let's remove the --infer_tool option completely.